src/uu/more/src/more.rs

```diff
@@ -255,6 +255,22 @@ fn more(buff: &str, stdout: &mut Stdout, next_file: Option<&str>, silent: bool)
                 }) => {
                     pager.page_up();
                 }
+                Event::Key(KeyEvent {
+                    code: KeyCode::Char('j'),
+                    modifiers: KeyModifiers::NONE,
+                }) => {
+                    if pager.should_close() {
+                        return;
```

Comment:
This caused a bug: it conflicts with the change in pull request #2767. This line now needs to `return Ok(());` or similar

Reply:
Fixed in https://github.com/uutils/coreutils/pull/2772